src/main/kotlin/Stylesheets.kt

```diff
@@ -0,0 +1,59 @@
+package com.dmdirc
+
+import com.sun.javafx.application.PlatformImpl
+import java.nio.file.*
+
+
+fun installStyles(file: Path) {
+    file.checkAndInstall()
+    val directory = file.toAbsolutePath().parent
+    directory.watchFile(file) {
+        PlatformImpl.setDefaultPlatformUserAgentStylesheet()
+        file.checkAndInstall()
+    }
+}
+
+private fun Path.checkAndInstall() {
+    if (Files.exists(this)) {
+        PlatformImpl.setPlatformUserAgentStylesheet(this.toAbsolutePath().toUri().toURL().toExternalForm())
+    }
+}
+
+private fun Path.watchFile(file: Path, action: () -> Unit) {
+    val watchPath = if (Files.isDirectory(this)) {
```

Comment:
I think this could be rewritten as this?:

```suggestion
    if (!Files.isDirectory(this)) return
    watch().takeRepeat { ....
```